src/test/java/edu/cornell/kfs/rass/batch/RassValueConverterBaseTest.java

```diff
@@ -76,6 +82,12 @@ public void testNoTruncate() {
         
     }
     
+    @Test
+    public void testRassLocalDateToSqlDateConverter() {
+        Date sqlDateResult = rassValueConverterBase.cleanDateValue(Award.class, "Budget Start Date", RassXmlAwardEntryFixture.SAMPLE_PROJECT.budgetStopDate);
+        assertTrue(RassXmlAwardEntryFixture.SAMPLE_PROJECT.budgetStopDate.toString().equalsIgnoreCase(sqlDateResult.toString()));
```

Comment:
I would suggest making this an `assertEquals()` check instead, especially since the date strings should only contain digits and dashes (thus making case insensitivity irrelevant). That way, if there is a mismatch, the test results should explicitly print the expected and actual string values for reference.